src/pages/transparency.tsx

```diff
@@ -48,9 +48,10 @@ export default function TransparencyPage() {
     <>
       <Navigation activeTab={NavigationTab.Transparency} />
       <Head
-        title={t('page.transparency.title') || ''}
-        description={t('page.transparency.mission.description') || ''}
+        title={t('page.transparency.title')}
+        description={t('page.transparency.mission.description')}
         image="https://decentraland.org/images/decentraland.png"
+        link={[{ rel: 'canonical', href: locations.transparency() }]}
```

Comment:
Shouldn't this prop be `links` as this can accept multiple ones?

Reply:
It is thus defined in Helmet's props, looking at the other props I think it refers to the name of the tag

![image](https://github.com/decentraland/governance/assets/45410089/de7365d9-40dc-4e93-905e-dd39f6b6bfeb)